specification/synapse/data-plane/Microsoft.Synapse/stable/2023-01-01/linkConnections.json

```diff
@@ -0,0 +1,1011 @@
+            "$ref": "./examples/LinkConnections_ListByWorkspace.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/linkconnections/{linkConnectionName}": {
+      "put": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_CreateOrUpdate",
+        "description": "Creates or updates a link connection",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "linkConnection",
+            "description": "Link connection resource definition",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LinkConnectionResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinkConnectionResource"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_CreateOrUpdate": {
+            "$ref": "./examples/LinkConnection_CreateOrUpdate.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_Get",
+        "description": "Get a link connection",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinkConnectionResource"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_Get": {
+            "$ref": "./examples/LinkConnection_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_Delete",
+        "description": "Delete a link connection",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Why do we have a 200 response in addition to the 204?

Reply:
When deleting LinkConnection, our service will call our dependent metadata service, which stores the LinkConnection info and just returns the response of it. And dependent metadata service changed behavior (200/204) of deleting metadata. So, to be compatible with two kinds of behavior, we would like to keep both 200 and 204.